test/supertest.js

```diff
@@ -150,7 +154,7 @@ describe('request(app)', function(){
 
     request(app)
     .get('/')
-    .expect(302, done);
+    .expect(302, function() { done(); });
```

Comment:
@jeffkole I am trying to understand the reason for this change (I see it was needed). 
How come this was required for only a redirect response?